### PR TITLE
Add keys for dependencies of Dropwizard

### PR DESCRIPTION
**Pull request by rhowe**
These are the keys (or lack thereof) for the current dependencies of the
Dropwizard project.

I haven't verified their authenticity. Caveat emptor.

### CODE REVIEW

**Comment by rhowe** [comment]
FYI @joschi & @jplock

File: resources/pgp-keys-map.list [hunk 1]
```diff
@@ -112,6 +112,17 @@ com.aoapps.platform             = 0x3FF945C37048D113E997AEC7CE70AA9A5BD10E01
...
+
+com.github.docker-java:docker-java-api = \
+                                  0x08B09F787B6745E796CE6DBE100306F2B3793BF3
+
+com.github.docker-java:docker-java-transport = \
+                                  0x08B09F787B6745E796CE6DBE100306F2B3793BF3
+
+com.github.docker-java:docker-java-transport-zerodep = \
+                                  0x08B09F787B6745E796CE6DBE100306F2B3793BF3
+
```
Comment (slawekjaranowski) [inline resources/pgp-keys-map.list:125]: The same key, please use group `com.github.docker-java = ....`

File: resources/pgp-keys-map.list [hunk 2]
```diff
@@ -207,6 +224,8 @@ com.pragmatickm                 = 0x3FF945C37048D113E997AEC7CE70AA9A5BD10E01
 
 com.puppycrawl.tools            = 0x06D34ED6FF73DE368A772A781063FE98BCECB758
 
+com.rabbitmq:amqp-client        = 0x0A9AF2115F4687BD29803A206B73A36E6026DFCA
```
Comment (slawekjaranowski) [inline resources/pgp-keys-map.list:227]: use group `com.rabbitmq=...`

File: resources/pgp-keys-map.list [hunk 3]
```diff
@@ -679,6 +719,8 @@ org.assertj                     = \
                                   0xA778FDE933A96DDA827840D8E2B3D84202B812D9, \
                                   0xBE685132AFD2740D9095F9040CC0B712FEE75827
 
+org.awaitility:awaitility       = 0x07E20F0103D9DFC697C490D0368557390486F2C5
```
Comment (slawekjaranowski) [inline resources/pgp-keys-map.list:722]: group `org.awaitility=...`

File: resources/pgp-keys-map.list [hunk 4]
```diff
@@ -878,6 +973,11 @@ org.nibor.autolink              = 0x624B96CEB9896889C97B258F7DC3076FE22D4F88
 
 org.objenesis                   = 0xE85AED155021AF8A6C6B7A4A7C7D8456294423BA
 
+org.openjdk.jmh:jmh-core        = 0x019082BC00E0324E2AEF4CF00D3B328562A119A7
+
+org.openjdk.jmh:jmh-generator-annprocess = \
+                                  0x019082BC00E0324E2AEF4CF00D3B328562A119A7
```
Comment (slawekjaranowski) [inline resources/pgp-keys-map.list:979]: group `org.openjdk.jmh=...`

File: resources/pgp-keys-map.list [hunk 5]
```diff
@@ -932,6 +1036,19 @@ org.springframework.*           = \
...
+org.testcontainers:jdbc         = 0xD33620F0D0864EC924483F6F3FF95B4331B1F3B7
+
+org.testcontainers:junit-jupiter = \
+                                  0xD33620F0D0864EC924483F6F3FF95B4331B1F3B7
+
+org.testcontainers:mysql        = 0xD33620F0D0864EC924483F6F3FF95B4331B1F3B7
+
+org.testcontainers:testcontainers = \
+                                  0xD33620F0D0864EC924483F6F3FF95B4331B1F3B7
+
```
Comment (slawekjaranowski) [inline resources/pgp-keys-map.list:1051]: group, the same key `org.testcontainers=...`

**Review by slawekjaranowski — CHANGES_REQUESTED** [review]
Very thanks for this PR. 

Please also add some examples of artifacts to test pom, when version will be upgraded we have sure that keys is still correct.

You can add to `pgp-keys-map-test1/pom.xml` or create separate module if you wish add more dropwizard artifacts.

**Comment by rhowe** [comment]
> Very thanks for this PR.
> 
> Please also add some examples of artifacts to test pom, when version will be upgraded we have sure that keys is still correct.
> 
> You can add to `pgp-keys-map-test1/pom.xml` or create separate module if you wish add more dropwizard artifacts.

Thanks. I've added entries for everything, except the google protobuf key is for an older version of the lib than is in the test pom, so I left that version as-is.

**Comment by slawekjaranowski** [comment]
@rhowe please fix build ...

**Comment by slawekjaranowski** [comment]
You can use `org.glassfish.jersey.*` the same key is used.

I don't see any dependency for `io.dropwizard...`

Maybe  we can use also `io.dropwizard.*` for keys

**Review by slawekjaranowski — APPROVED** [review]

**Pull request merged**